arm/Microsoft.AppConfiguration/configurationStores/readme.md

```diff
@@ -0,0 +1,86 @@
+# App Configuration `[Microsoft.AppConfiguration/configurationStores]`
+
```

Comment:
Add short description. By default we use something like: `This module deploys a App Configuration Configuration Store`. However, it can be more than that.